### PR TITLE
SN Add properties to library

### PR DESCRIPTION
**Pull request by sarahgonicholson**
* Add properties `target_dna_sequence`, `guide_rna_sequence`, and `antibody` to `Library` schema
* Change property `restriction_enzymes` to `enzymes`  in `LibraryPreparation` (recently added property so it is empty in the portal) to apply for transposase enzymes as well and update associated tests.

### CODE REVIEW

File: src/encoded/schemas/library.json [hunk 1]
```diff
@@ -95,6 +95,11 @@
             "type": "number",
             "minimum": 0
         },
+        "antibody": {
```
Comment (aschroed) [inline src/encoded/schemas/library.json:98]: May want to add some enum or suggested enum here or you are likely to get a wide variety of values input.  In 4DN we have an antibody item and the 'title' is a calcprop that combines target and vendor.  If an Item is not needed (it likely isn't) then you may want to set up some rules for naming conventions.

**Review by aschroed — APPROVED** [review]
I left a couple of comments for your consideration but non-blocking.

File: src/encoded/schemas/library.json [hunk 2]
```diff
@@ -125,6 +130,11 @@
             "type": "number",
             "minimum": 0
         },
+        "guide_rna_sequence": {
+            "title": "Guide RNA Sequence",
+            "description": "Guide RNA sequence. Relevant for Cas9 Nanopore",
```
Comment (wfeng19) [inline src/encoded/schemas/library.json:135]: Maybe change description to "Relevant for Cas9 capture experiments"?

File: src/encoded/schemas/library.json [hunk 3]
```diff
@@ -155,6 +165,11 @@
             "type": "string",
             "format": "date"
         },
+        "target_dna_sequence": {
+            "title": "Target DNA Sequence",
+            "description": "Target DNA sequence used in insert selection",
```
Comment (wfeng19) [inline src/encoded/schemas/library.json:170]: Was this property also intended for the Cas9 capture experiments? It looks like we added a "dna_target" property on the UMich metadata sheet (https://docs.google.com/spreadsheets/d/1sFOEEc_JtE-wNO7RnN4EcaEVeBuDU8uMNHnAJzoLc2s/edit?gid=493168297#gid=493168297), but I wasn't sure if this property was intended to be used in the same way.

**Review by wfeng19 — COMMENTED** [review]
I left a couple of comments, but nothing non-blocking as well.

File: src/encoded/schemas/library.json (hunk 3 above)
Comment (sarahgonicholson) [reply src/encoded/schemas/library.json:170]: I see, I wasn't sure exactly what the intended values were for this. Updating!

**Pull request merged**